### PR TITLE
Use HTTPS where possible

### PR DESCRIPTION
**Pull request by mathiasbynens**
This avoids the mixed content warnings on https://fortawesome.github.io/Font-Awesome/.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,31 +23,24 @@
...
     _gaq.push(['_trackPageview']);
 
     (function() {
-      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
+      var ga = document.createElement('script');
+      ga.src = 'https://ssl.google-analytics.com/ga.js';
+      var s = document.scripts[0];
+      s.parentNode.insertBefore(ga, s);
```
Comment (mathiasbynens) [inline index.html]: Cleaned up as per <https://mathiasbynens.be/notes/async-analytics-snippet>.

**Comment by tagliala** [comment]
@mathiasbynens could you please also fix the master branch?

**Comment by mathiasbynens** [comment]
@tagliala Will do!

**Comment by mathiasbynens** [comment]
#6104

**Comment by tagliala** [comment]
closing here

**Pull request closed without merging**